MAINTAINING.md

```diff
@@ -0,0 +1,83 @@
+## Adding a new model
+
+TLDR; update the the `WEIGHT_LOOKUP` mapping and then run `make publish_models`.
+
+#### Weight lookup mapping
+
+The weight lookup mapping in `zamba/models/config.py` connectss each model to the private s3 directory where model training results live. For example,
```

Comment:
Typo "connectss"

Reply:
Maybe note that the "private" bucket is only used as the source of what to publish. It is not used by any of the user-facing models (all their weights come from the public buckets).